source/docs/advanced/theme/sitemap.md

```diff
@@ -0,0 +1,172 @@
+---
+id: sitemap
+title: Configure the sitemap
+---
+
+> **Prequisite:** To follow this guide, you first need to know [how to create new pages](/docs/essentials/add-a-page-client-side.html) and [a GraphQL module](/docs/essentials/extend-the-graphql-schema.html) in Front-Commerce.
+
+The sitemap is a key piece of each e-commerce website. It allows search engines to crawl and index your pages faster, driving to more visitors and potential clients.
+
+## Generate a Sitemap
+
+Within Front-Commerce you can generate a sitemap by running the command `npm run sitemap` in your Skeleton. It will run a GraphQL query that will fetch all the routes registered in your application.
```

Comment:
```suggestion
Within Front-Commerce you can generate a sitemap by running the [`npm run sitemap` command](/docs/reference/scripts.html#sitemap-js) from your Skeleton. It will run a GraphQL query that will fetch all the routes registered in your application.
```